bin/node/Cargo.toml

```diff
@@ -25,48 +26,48 @@ hex = "0.4"
-sp-timestamp = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.19" }
-pallet-staking = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.19" }
+sp-application-crypto = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sc-block-builder = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sc-cli = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23", features = ["wasmtime"]}
+sp-core = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sc-executor = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23", features = ["wasmtime"]}
+sc-service = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23", features = ["wasmtime"]}
+sc-telemetry = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sc-keystore = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sp-keystore = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sp-inherents = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sc-basic-authorship = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sc-network = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sc-transaction-pool = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sp-transaction-pool = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sc-transaction-pool-api = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sc-consensus-aura = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sp-consensus-aura = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sp-consensus = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sc-consensus = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sc-client-api = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23"}
+sp-timestamp = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23" }
+pallet-staking = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.23" }
 
 aleph-runtime = { path = "../runtime"}
 finality-aleph = { path = "../../finality-aleph"}
 aleph-primitives = { package = "primitives", path = "../../primitives"}
 
 # These dependencies are used for the node's RPCs
+jsonrpsee = { version = "0.13.0", features = ["server"] }
 jsonrpc-core = "18.0"
```

Comment:
Is this still needed after switching to `jsonrpsee`?

Reply:
Done